src/main/java/dal/fatih/todoly/TaskRepository.java

```diff
@@ -0,0 +1,164 @@
+    protected Connection connection = dbConnection.getConnection();
+    final PreparedStatement createPrepareStatement;
+    final PreparedStatement listPrepareStatement;
+    final PreparedStatement getPreparedStatement;
+    final PreparedStatement deleteTaskByIdPreparedStatement;
+    final PreparedStatement filterPreparedStatement;
+    final PreparedStatement getByTitleOrDesPreparedStatement;
+    private List<Task> tasks;
+
+    public TaskRepository() throws SQLException {
+
+        createTable();
+        createPrepareStatement =
+                connection.prepareStatement("insert into tasks " + "values (?,?,?,?,?)");
+        listPrepareStatement =
+                connection.prepareStatement("select id,taskid,title,description,duedate from tasks");
+        getPreparedStatement =
+                connection.prepareStatement("select taskid,title,description,duedate from tasks where taskid=? limit 1");
+        deleteTaskByIdPreparedStatement =
+                connection.prepareStatement("delete from tasks where taskid=?");
+        filterPreparedStatement =
+                connection.prepareStatement("select * from tasks where duedate between now() and ?");
+        getByTitleOrDesPreparedStatement =
+                connection.prepareStatement("select taskid,title,description,duedate from tasks where title like ? "
+                        + "or description like ?");
+    }
+
+    public void clearTable() {
+        try {
+            PreparedStatement preparedStatement = connection.prepareStatement("TRUNCATE TABLE tasks");
+            preparedStatement.execute();
+            connection.close();
+            preparedStatement.close();
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void createTable() {
+        try {
+            PreparedStatement preparedStatement = connection.prepareStatement("create table if not exists tasks"
+                    + "  (id int auto_increment primary key not null,"
+                    + "   taskid uuid(36),"
+                    + "   title varchar(35),"
+                    + "   description varchar(250),"
+                    + "   duedate date)");
+            preparedStatement.executeUpdate();
+            preparedStatement.close();
+        } catch (SQLException exception) {
+            System.out.println("Check database connections, drivers!!!");
+        }
+    }
+
+    public void create(Task task) {
+        try {
+            createPrepareStatement.setString(1, null);
+            createPrepareStatement.setString(2, task.getId().toString());
+            createPrepareStatement.setString(3, task.getTitle());
+            createPrepareStatement.setString(4, task.getDescription());
+            createPrepareStatement.setString(5, task.getDueDate().toString());
+            createPrepareStatement.executeUpdate();
+
+            System.out.println("\n" + task.getId() + " Task added");

```

Comment:
Do not print anything in this class. Assume this class to be only responsible for DB access for task objects